dbms/src/Columns/ColumnVector.h

```diff
@@ -229,6 +229,11 @@ class ColumnVector final : public COWPtrHelper<ColumnVectorHelper, ColumnVector<
             data[i + old_size] = src_container[position_vec[i]];
     }
 
+    void insertMany(const Field & field, size_t length) override
+    {
+        data.resize_fill(data.size() + length, static_cast<T>(field.get<T>()));
+    }
+
```

Comment:
/cc @breezewish vector may need to pick it back to the cse release-7.5 branch

Reply:
`insertMany` is a new interface. I misunderstand it. We don't need to backport this in the cse branch.

Reply:
Just curious,

> I used insertManyFrom in https://github.com/pingcap/tiflash/pull/8854, where the source column is a const column while the target column is not and it causes https://github.com/pingcap/tiflash/issues/8904.

Why insertManyFrom causes issues?

Reply:
Actually, it is caused by calling `ColumnVector::insertManyForm(ColumnConst, ... )`. So `static_cast<const Self &>(column_const)` just returns a random memory address and introduces bugs.

https://github.com/pingcap/tiflash/blob/8cec573af26adf5d3d974506ac4e9cbaecd7d5f6/dbms/src/Columns/ColumnVector.h#L216-L220